{"name": "Test to validate when user complete the task without completing the self verification", "status": "passed", "description": "This testcase validate error message when user complete the task without completing the self verification", "parameters": [{"name": "job_execution_by", "value": "'user'"}, {"name": "users_list", "value": "['process.publisher.03', 'facility.admin.03']"}], "start": 1733762204484, "stop": 1733762220956, "uuid": "d477ee38-8fd1-49f2-8584-e61308c55d1a", "historyId": "bb5cb6ad8e2e93ad06b6c418756c9ddc", "testCaseId": "7179230d6a164f8c79500c2c19ec3cda", "fullName": "tests.scenarios.job.test_parameter_verification.TestParameterVerification#test_validate_error_on_not_completing_self_verification", "labels": [{"name": "tag", "value": "validate_error_on_not_completing_self_verification"}, {"name": "tag", "value": "parameter_verification"}, {"name": "tag", "value": "job"}, {"name": "tag", "value": "scenario"}, {"name": "parentSuite", "value": "tests.scenarios.job"}, {"name": "suite", "value": "test_parameter_verification"}, {"name": "subSuite", "value": "TestParameterVerification"}, {"name": "host", "value": "ubuntu"}, {"name": "thread", "value": "53959-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.scenarios.job.test_parameter_verification"}], "links": [{"type": "link", "url": "https://app.clickup.com/t/86ctxyyau", "name": "https://app.clickup.com/t/86ctxyyau"}]}